### PR TITLE
add centerView button in IDE

### PR DESCRIPTION
**Pull request by Bouh**
This add "Center view" button in IDE with the other button for Zooming

### CODE REVIEW

File: newIDE/app/src/InstancesEditor/index.js [hunk 1]
```diff
@@ -578,6 +578,12 @@ export default class InstancesEditorContainer extends Component {
     this.viewPosition.scrollTo(x, y);
   }
 
+  centerView() {
+    const x = this.props.project.getMainWindowDefaultWidth() / 2 || 0;
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/index.js]: The || 0 is unnecessary, `getMainWindowDefaultWidth()` is guaranteed to return a number

File: newIDE/app/src/SceneEditor/Toolbar.js [hunk 2]
```diff
@@ -138,6 +138,10 @@ export class Toolbar extends PureComponent {
               click: this.props.zoomOut,
               accelerator: 'CmdOrCtrl+-',
             },
+            {
+              label: 'Center view',
```
Comment (4ian) [inline newIDE/app/src/SceneEditor/Toolbar.js]: Should that be "Center View" or something like "Go back to origin"?

**Comment by 4ian** [comment]
Don't forget Prettier ;)

**Comment by Bouh** [comment]
> Don't forget Prettier ;)

omg always :') i need to add this command before push my code

**Comment by 4ian** [comment]
Looks good! Thanks for adding this! Will release a version this week-end.

**Pull request merged**